pkg/minikube/shell/shell.go

```diff
@@ -125,6 +125,17 @@ REM @FOR /f "tokens=*" %%i IN ('%s') DO @%%i
 `, s...)
 		},
 	},
+	"tcsh": {
```

Comment:
Maybe add a matching test case ?

Reply:
Took quite the rewrite haha. Done.

Reply:
I just meant a regular (unit) test, but adding a functional test is probably a good addition as well...

https://github.com/kubernetes/minikube/blob/fabda37a91f262355b3250373efd603ac5edf334/pkg/minikube/shell/shell_test.go

Reply:
Done.